### PR TITLE
Submit project

### PR DESCRIPTION
**Pull request by Jones-Git**
Here's my project, I hope you like it.

### CODE REVIEW

**Comment by Jones-Git** [comment]
The Extra credit I have done is the one that calls for Additional pages beyond the first one, and some additional CSS beyond what was required.

File: project_one_Brian-Jones/index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <link rel="stylesheet" href="style.css">
+    <title>Placeholder</title>
+
+</head>
+
+<body>
+    <h1>Header for big things that are to follow</h1>
+
+    <div>
+        <p>Press me to make back things happen: <button id="grading-element-one" onclick="baddie()">DO IT</button></p>
```
Comment (lehmanclass) [inline project_one_Brian-Jones/index.html]: works as expected

File: project_one_Brian-Jones/index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        
+    </div>
+
+    <div id="container">
+
+        <a href="/about.html" target="_blank"><button id="grading-element-two">About page </button></a>
```
Comment (lehmanclass) [inline project_one_Brian-Jones/index.html]: works as expected

File: project_one_Brian-Jones/index.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+        <p>Click me to fill something out <button id="grading-element-three" onclick="unHide()">Press me</button></p>
+
+
+        <div id="grading-element-three-consequence" hidden>
```
Comment (lehmanclass) [inline project_one_Brian-Jones/index.html]: almost works as expected. right now you are changing `display: none` to `display: block`...can you make it start as `visibility: hidden` to `visibility: visible`

File: project_one_Brian-Jones/index.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+            <form action="http://localhost:3000/form" method="POST" id="grading-element-four">
```
Comment (lehmanclass) [inline project_one_Brian-Jones/index.html]: almost there! the form submits `testOne, testTwo, testThree` as expected...but has no values. Can you use javascript to put a non-empty value in all three?

File: project_one_Brian-Jones/index.html [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+                            </tr>
+
+                        </tbody>
+                    </table>
+                </fieldset>
+            </form>
+        </div>
+
+        <div>
+            <h1 id="grading-element-five">Hover over me to do a thing.</h1>
```
Comment (lehmanclass) [inline project_one_Brian-Jones/index.html]: works as expected

**Comment by lehmanclass** [comment]
Very close. Can you put your email in the description, and any extra credit in a `README.md` file?

**Comment by Jones-Git** [comment]
Had to fix some errors.

My email is: Jones-Jamal@outlook.com

Lehman email: Brian.Jones@lc.cuny.edu

**Pull request closed without merging**